lib/resty/core/response.lua

```diff
@@ -161,8 +167,7 @@ local function get_resp_header(tb, key)
         return ret
     end
 
-    -- n == FFI_ERROR
-    error("no memory")
+    error("unreachable")
```

Comment:
Maybe better to follow the existing `set_resp_header` pattern (because already existing) and assume that the default case is `FFI_ERROR`? As in: https://github.com/openresty/lua-resty-core/blob/42d1d948dc4de045810f7a213b7f76aa206034ed/lib/resty/core/response.lua#L116-L117

Reply:
@thibaultcha 
Updated.